<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="szukaj">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Szukaj:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="przeglad_ksiazki">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Przeglądaj książki</string>
    </property>
   </widget>
   <widget class="QPushButton" name="przeglad_users">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Przeglądaj użytkowników</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dodaj_konto">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj konto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dodaj_ksiazke">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj książkę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wypozycz_ksiazke">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wypożycz książkę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zwroc_ksiazke">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zwróć książkę</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wypozyczone_ksiazki">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wypożyczone książki</string>
    </property>
   </widget>
   <widget class="QLabel" name="bledy">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Systemu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBiblioteka">
    <property name="title">
     <string>Biblioteka</string>
    </property>
   </widget>
   <addaction name="menuBiblioteka"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
